scripts/shared/clusters.sh

```diff
@@ -36,7 +37,7 @@ function generate_cluster_yaml() {
     local service_cidr="${service_CIDRs[${cluster}]}"
     local dns_domain="${cluster}.local"
     local disable_cni="true"
-    if [[ "${cluster}" = "cluster1" ]]; then
+    if [[ "${cluster}" = "${clusters[0]}" ]]; then
```

Comment:
Please leave as is, we'll handle broker cluster in a different PR

Reply:
Yes for now, we'll take care of broker cluster better in a different PR

This is tangential to your PR anyhow..

Reply:
Well not really b/c I'm also trying to remove hard-coded cluster names. There's not much purpose in allowing the user to specify the cluster names if we're going to assume certain names anyway 😄  I'd rather be complete and I don't see the harm in changing these references now. I mean I don't see why it would matter to a subsequent PR (if/whenever that lands) if this is referenced as "cluster1" or "${clusters[0]}".